### ISSUE TITLE
Option showWeekNumbers shows 36th week twice

### ISSUE THREAD

**Issue by deasko** [gpbl/react-day-picker#1399]
### Bug description

When active option `showWeekNumbers`, the 36th week appears duplicated.

### To reproduce 

Code sandbox: https://codesandbox.io/s/react-day-picker-examples-customization-week-numbers-forked-pb4y6?file=/example.js

#### Steps

1. Create a DayPicker and set it to Sep 2021.
2. Notice the 36th week twice

### Expected behavior

Show the 36th week only once.

### Screenshots

<img width="273" alt="Cap 2022-02-08 a la(s) 15 33 07" src="https://user-images.githubusercontent.com/1635160/153052933-ab0bcbb8-0df3-4bd5-9006-b5fbfc38bd09.png">

**Comment by Go1dExperience** [comment]
Hi, I'm actually not seeing what you described. Perhaps this is a problem only on your side?
![Screenshot 2022-02-09 172720](https://user-images.githubusercontent.com/28619426/153179630-7338f7ef-9b87-4074-ac9b-16438c3ef353.png)

**Comment by deasko** [comment]
Umh that's strange. On codesandbox I see the problem, notice 36th week appears for last week of august, then first and second weeks of september. 

<img width="1060" alt="Cap 2022-02-09 a la(s) 09 53 04" src="https://user-images.githubusercontent.com/1635160/153205036-8d93b402-2562-45c5-8ecd-dbb4e78d413e.png">

<img width="1037" alt="Cap 2022-02-09 a la(s) 09 52 47" src="https://user-images.githubusercontent.com/1635160/153204949-8d4c12f4-2fbc-47de-92fd-e0a285e45ca3.png">


> Hi, I'm actually not seeing what you described. Perhaps this is a problem only on your side? ![Screenshot 2022-02-09 172720](https://user-images.githubusercontent.com/28619426/153179630-7338f7ef-9b87-4074-ac9b-16438c3ef353.png)

**Comment by deasko** [comment]
Any updates for this?

**Comment by gpbl** [comment]
No updates @deasko, since I'm currently working on the next version which will fix this issue too! Thanks for your patience!

**Comment by gpbl** [comment]
Closing as [we've released v8](https://github.com/gpbl/react-day-picker/discussions/1412): this issue belongs to a version which is not longer maintained, and it is likely fixed. In case, please open a new issue. Thanks!
